Documentation/ceph-dashboard.md

```diff
@@ -40,6 +40,8 @@ If you are on a node in the cluster, you will be able to connect to the dashboar
 DNS name of the service at `https://rook-ceph-mgr-dashboard-https:8443` or by connecting to the cluster IP,
 in this example at `https://10.110.113.240:8443`.
 
+Please note the dashboard will only be enabled for the first ceph object store.
```

Comment:
So that it shows up a little bigger to readers:

```suggestion
> **IMPORTANT:** Please note the dashboard will only be enabled for the first Ceph object store created by Rook.
```